api/apps/articles/model.coffee

```diff
@@ -390,7 +390,7 @@ typecastIds = (article) ->
     super_article: if article.super_article?.related_articles then _.extend article.super_article, related_articles: article.super_article.related_articles.map(ObjectId) else {}
 
 @sendArticleToSailthru = (article, cb) =>
-  cb null unless NODE_ENV is 'production'
+  return cb() unless NODE_ENV is 'production'
```

Comment:
Whoops! Missed that, good catch—always remember to `return callback` to short circuit functions.